src/coverdb.rs

```diff
@@ -81,15 +83,13 @@ impl CoverDB {
             
             let id: i32 = Cover::table.select(Cover::ID).order(Cover::ID.desc()).first(&self.conn).context("Unable to get ID of cover")?;
             // the database fields needs to be uncompressed/non-overlapping
-            let thumbnail: Vec<u8> = image.resize_to_fill(TEXTURE_WIDTH as u32, TEXTURE_HEIGHT as u32, FilterType::Triangle).into_rgb()
-                .pixels().flat_map(|pixel| pixel.channels()).copied().collect();
             diesel::insert_into(CoverThumbnail::table)
                 .values((
                     CoverThumbnail::ID.eq(id),
                     CoverThumbnail::Format.eq(TEXTURE_FORMAT),
-                    CoverThumbnail::Width.eq(TEXTURE_WIDTH),
-                    CoverThumbnail::Height.eq(TEXTURE_HEIGHT),
-                    CoverThumbnail::Data.eq(Some(&thumbnail)),
+                    CoverThumbnail::Width.eq(image.width() as i32),
```

Comment:
Sure? Usdx code seems to store always the same value here.

Reply:
When tested the stored size was always the same for the thumbnail and the cover